low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -243,16 +248,16 @@ class SSPLDevDeploy:
 
     def setup_kafka(self):
         """Setup kafka"""
-        file_path = (f"{self.CORTX_MONITOR_BASE_URL}"
-                     "/low-level/files/opt/seagate/sspl/setup/standalone")
-        kafka_version = '2.13-2.7.0'
+        kafka_version = '2.13_2.7.0'
```

Comment:
Shouldn't the Kafka version be coming from some config rather then being hard coded here?

Reply:
yes, but since this is dev_deploy script, the SSPL won't be installed yet right, these are prereq steps, that's why it is hardcoded I guess. @mariyappanp can we improve it?

Reply:
As RPM is available now for kakfa and zookeeper, cant we use that directly ?

Reply:
yes @sandeepsgt, installing the same

Reply:
eliminated version number variable, please check @jjxsg @sandeepsgt